### ISSUE TITLE
hap encoder 1.0.8-rc4 won't uninstall

### ISSUE THREAD

**Issue by onkelherzog** [disguise-one/hap-encoder-adobe-cc#30]
Hallo folks.
I wanted to uninstall the encoder due to issues with Premiere 2019 (export window won't open with the plugin installed, only 2018 works).
It says "uninstall failed". When I try to uninstall via Apps & Features.

How can I do a clean uninstall of the encoder? 

Thanks for you reply!

**Comment by GregBakker** [comment]
Sorry to hear about the problem!

The plugin installs to
    C:\Program Files\Adobe\Common\Plug-ins\7.0\MediaCore\HapEncoderPlugin.prm
If that file's locked by some other process then the uninstall would fail.

Which version of the plugin were you using by the way? Would like to fix the issue if possible.

Speculation: something went wrong during an export in Premiere 2019; and it hung on exit - keeping the HapEncoderPlugin.prm locked so it couldn't be uninstalled. Another thing I've seen is that some importers (not this plugin, it's only doing export) can cause the export window lockup in AME if broken .mov files are present in the watch directories.

Please let me know if you have any more details - AFAIK the 1.0.9-rc1 alpha version of the plugin at https://github.com/disguise-one/hap-encoder-adobe-cc/releases is working ok in Premiere 2019.

Greg

**Comment by onkelherzog** [comment]
Hi Greg,

thanks for getting back to me.
It is like you say. If I uninstall the program via apps & features or via
the uninstaller within the folder the plugin is "stuck" in the common
folder" That's why the path of the plugin is still active in apps &
features.

I'm using the 1.0.8-rc4 because at home hap export worked with cc 2018 and
we had a deadline.
At work and at home I couldnt get the 2019 version to export haps. Thats
why I went this way.

As of right now if have both the 2018 and 2019 installed. Could there be a
conflict?

I tried the 1.0.9 version previously. It's not any different.

I'll try and do a clean install of Adobe Suite at work on tuesday and will
try the 1.0.9-rc1 alpha on 2019 Adobe only.

Windows 1809

Thanks for your help.
